### ISSUE TITLE
CIHP dataset in tutorial only have 12 classes instead of 20

### ISSUE THREAD

**Issue by scrouzet** [keras-team/keras-io#1227]
I am referring to the example https://keras.io/examples/vision/deeplabv3_plus/
The corresponding github is https://github.com/keras-team/keras-io/blob/master/examples/vision/deeplabv3_plus.py

It is specified that the CIHP dataset has 20 classes (19 classes such as Hair, Dress, etc + background).

However, when I look at the png files corresponding to the masks, there is only 12 different values: [  0   9  29  55  59  75  76  95 126 140 178 210] and not 20.

I found the same set of values in the Training and Validation sets. So it seems that this dataset does not really include what is refered to.

**Comment by scrouzet** [comment]
This was my mistake, the CIHP dataset is all fine. The issue was that I was looking at the directory 'Categories' and not 'Category_ids'.
My apologies for the unnecessary opening of a new issue.
